.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+	"name": "Node.js",
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:1-20-bookworm",
```

Comment:
Can Dependabot be configured to keep this up to date?

Reply:
@luc122c Yeah, but only for devcontainer features, not for the base image.

Reply:
@FloEdelmann Dependabot can't update the base Docker image. However, we could change the referenced image to:
```
mcr.microsoft.com/devcontainers/javascript-node:20
```
The image tag (`20`) references the Node.js major version. Docker will automatically pull the latest minor/patch releases. This means you only need to update the image tag when you increase the major Node.js version of OFL, which won't happen often.